src/components/hack-details.js

```diff
@@ -64,66 +59,74 @@ export function HackDetails({ match, dispatch, userId, hackDetails, history }) {
             alignItems="stretch"
             alignContent="center"
           >
-            <Grid item xs={9}>
-              <HackStatus status={hackDetails.status}/>
-            </Grid>
+            <Paper className={classes.root}>
```

Comment:
You should extract this into a `ViewHack` component to make this component only concerned with loading the hack and routing to the correct view.

Reply:
That is the plan!